templates/glue-job-compare-parquet.yaml

```diff
@@ -0,0 +1,86 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This cf template ended up being an exact copy of `glue-job-S3ToJsonS3.yaml`

Reply:
+1 to Phil